### ISSUE TITLE
Holodex brand identity design

### ISSUE THREAD

**Issue by ahdinosaur** [open-app/holodex#50]
deliverables:
- [ ] logo
- [ ] landing page design
- [ ] app style guide
- ...?

value: we can communicate Holodex beyond words.

**Comment by rdbartlett** [comment]
re: app style guide: I would like to recommend in the most hearty terms that y'all give the Google Material design spec a really good look. The more I explore it, the more I appreciate it. Happy to have a yarn sometime about how much fun I'm having with Material + Sketch + Invision as my design stack

https://www.google.com/design/spec/material-design/introduction.html

**Comment by ahdinosaur** [comment]
sweet, thanks for the recommendation @rdbartlett. :smile_cat: 

i have in mind more feelings over functionality at the moment, so resources that reflect my feelings for Holodex:
- [my 'Life as a Holon' post](https://blog.dinosaur.is/life-as-a-holon/)
- ['Hive' video](https://www.youtube.com/watch?v=0pdTi1ui3I4)
- [n-flake](https://en.wikipedia.org/wiki/N-flake)

**Comment by ahdinosaur** [comment]
as used in @nanz's [first design](https://camo.githubusercontent.com/da769e612f445a5d4978c5542ca622692e1f7857/687474703a2f2f692e696d6775722e636f6d2f6d4d656871526a2e706e67), i think purple is a really great color, i'm keen to use it more. :purple_heart:

**Comment by rdbartlett** [comment]
Yeah purple is great. Here's what Material has to say about color :) 

https://www.google.com/design/spec/style/color.html#

**Comment by simontegg** [comment]
Bro, one step ahead   :) https://github.com/holodex/app/blob/master/src%2Fnode_modules%2Fui%2Fvars.js#L75
